hobbyloop-api/src/main/java/hobbyloop/backend/api/controller/ticket/UserTicketInfoListResponseDTO.java

```diff
@@ -0,0 +1,28 @@
+package hobbyloop.backend.api.controller.ticket;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import hobbyloop.backend.domain.ticket.UserTicket;
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+
+@Getter
+@NoArgsConstructor(access = AccessLevel.PROTECTED)
+@AllArgsConstructor
+@Builder
+public class UserTicketInfoListResponseDTO {
+	private String centerName;
+	private List<UserTicketInfoResponseDTO> ticketInfos;
+
+	public static UserTicketInfoListResponseDTO from(String centerName, List<UserTicket> userTickets) {
+
+		return UserTicketInfoListResponseDTO.builder()
+			.centerName(centerName)
+			.ticketInfos(userTickets.stream().map(UserTicketInfoResponseDTO::from).collect(Collectors.toList()))
```

Comment:
```suggestion
List<UserTicketInfoResponseDTO> userTicketInfos = userTickets.stream()
.map(UserTicketInfoResponseDTO::from)
.collect(Collectors.toList())
         ...

                         .centerName(centerName)
			.ticketInfos(userTicketInfos)
```

너무 우겨넣은 느낌이라 별도 변수로 빼서 넣어주면 더 좋을 것 같습니다

Reply:
굳이 바꾸지는 않아도 됩니다 

나중에 에러 / 디버깅 로깅에서는 별도로 빼줘야 편할 때가 있어서 코멘트 남깁니다

Reply:
너무 로직 하나 내에서 많은걸 하려고 하는 느낌이라는 말인가요???

Reply:
파람으로 들어가는 곳에 변수가 아닌, 결국 메서드 체이닝으로 변수를 생성해서 넣다 보니 코드적으로도 좋지 않고 
직관적으로 뭘 넣는 지 안보인단 얘기였습니다. 
추가적으로 로깅/디버깅은  method 내에 변수 넣기전에 해당 변수로 로깅찍을 수 있단 말이었습니다

Reply:
아 어떤 말인지 이해했습니다
수정 내용에 반영하겠습니다